projects/demo/src/modules/components/table/examples/4/index.html

```diff
@@ -55,8 +55,7 @@
         [columns]="columns"
         [direction]="(direction$ | async) || 1"
         [tuiSortBy]="sorter$ | async"
-        (directionChange)="direction$.next($event)"
-        (tuiSortByChange)="sorter$.next($event!)"
+        (tuiSortAndOrderChange)="change($event)"
```

Comment:
Let's rename it to `tuiSortChange` for brevity?

Reply:
fixed